packages/conventional-changelog-conventionalcommits/conventional-recommended-bump.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const parserOpts = require(`./parser-opts`)
+
+module.exports = function (config) {
+  return {
+    parserOpts: parserOpts(config),
+
+    whatBump: (commits) => {
+      let level = 2
```

Comment:
What is our stance on releasing on every change? (I maintain my own personal changelog preset solely for the purpose of only recommending a release on fixes, features, and breaking changes.)

Reply:
I would like to continue releasing changes for every commit type; this is how the angular preset approaches things, and is what I myself (and I expect other users of `standard-version`) have come to expect.

Reply:
I suppose we could open this up to being configured? it feels really weird to me though as a library maintainer. If I've performed a maintenance task, such as upgrading libraries, or performance refactoring, etc., I still often want to release a patch release of my module -- I have never personally found myself in a position where the default behavior of bumping the `PATCH` wasn't the behavior I wanted.

Reply:
> I suppose we could open this up to being configured?

As a first pass, let's please not let my comment hold up your PR. :sweat_smile: 

Long-term, yes, I would really like to make releases configurable. My own preset was actually a pre-cursor to the one I wrote for my company because the number of releases for tasks, such as chores related to project configuration, refactors related to reducing technical debt, and docs related to inline comments, were becoming disruptive to our library consumers :disappointed:  (new releases caused our dependency update tool to roll that out to hundreds of applications in near real-time, which caused lots of notifications, CI job queues to grow, etc.).

> If I've performed a maintenance task, such as upgrading libraries, or performance refactoring

Those sound like tasks that effect downstream consumers. Those, in my opinion, should be released. :+1:

Reply:
I understand this use-case better now; let's take the conversation to another issue?